main/iam-policy-secretsmanager.tf

```diff
@@ -0,0 +1,60 @@
+/**
+ * Copyright (C) 2018-2019 Expedia Inc.
```

Comment:
Why do some license files say 2018-2019 and same say 2019?

Reply:
fixed